controls/V-71859.rb

```diff
@@ -1,118 +1,113 @@
+  tag "rid": "SV-86483r3_rule"
   tag "stig_id": "RHEL-07-010030"
-  tag "cci": "CCI-000048"
+  tag "cci": ["CCI-000048"]
+  tag "documentable": false
   tag "nist": ["AC-8 a", "Rev_4"]
   tag "check": "Verify the operating system displays the Standard Mandatory DoD
-  Notice and Consent Banner before granting access to the operating system via a
-  graphical user logon.
+Notice and Consent Banner before granting access to the operating system via a
+graphical user logon.
+
+Note: If the system does not have GNOME installed, this requirement is Not
+Applicable.
+
+Check to see if the operating system displays a banner at the logon screen with
+the following command:
 
-  Note: If the system does not have GNOME installed, this requirement is Not
-  Applicable.
+# grep banner-message-enable /etc/dconf/db/local.d/*
+banner-message-enable=true
 
-  Check to see if the operating system displays a banner at the logon screen with the
-  following command:
+If \"banner-message-enable\" is set to \"false\" or is missing, this is a
+finding."
+  tag "fix": "Configure the operating system to display the Standard Mandatory
+DoD Notice and Consent Banner before granting access to the system.
 
-  # grep banner-message-enable /etc/dconf/db/local.d/* banner-message-enable=true
+Note: If the system does not have GNOME installed, this requirement is Not
+Applicable.
 
-  If \"banner-message-enable\" is set to \"false\" or is missing, this is a finding."
+Create a database to contain the system-wide graphical user logon settings (if
+it does not already exist) with the following command:
 
-  tag "fix": "Configure the operating system to display the Standard Mandatory DoD
-  Notice and Consent Banner before granting access to the system.
+# touch /etc/dconf/db/local.d/01-banner-message
 
-  Note: If the system does not have GNOME installed, this requirement is Not
-  Applicable.
+Add the following line to the [org/gnome/login-screen] section of the
+\"/etc/dconf/db/local.d/01-banner-message\":
 
-  Create a database to contain the system-wide graphical user logon settings (if it
-  does not already exist) with the following command:
+[org/gnome/login-screen]
+banner-message-enable=true
 
-  # touch /etc/dconf/db/local.d/01-banner-message
+Update the system databases:
 
-  Add the following line to the [org/gnome/login-screen] section of the
-  \"/etc/dconf/db/local.d/01-banner-message\":
+# dconf update
 
-  [org/gnome/login-screen]
-  banner-message-enable=true"
+Users must log out and back in again before the system-wide settings take
+effect."
+  tag "fix_id": "F-78211r4_fix"
 
   only_if { command('dconf').exist? or file('/etc/gdm/custom.conf').exist? }
 
-  describe command("dconf read /org/gnome/login-screen/banner-mesage-enable") do
-    its('stdout'.strip) { should eq 'true'}
+  describe command("dconf read /org/gnome/login-screen/banner-message-enable") do
+    its('stdout.strip.to_s') { should cmp BANNER_MESSAGE_ENABLED.to_s }
```

Comment:
Use of global variables can cause issues during test runs, please either move this into `libraries` or change to a local variable.

Reply:
White space should be ignored when comparing the message since users may have changed the spacing to comply with local accessibility requirements.